src/libs/actions/BankAccounts.js

```diff
@@ -179,6 +179,15 @@ function deletePaymentBankAccount(bankAccountID) {
                 value: {[bankAccountID]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}},
             },
         ],
+
+        // Delete the bank account locally after a successful request
```

Comment:
```suggestion
        // On deleting bank account offline when we reconnect, the onyx data isn't getting updated
        // so we are setting it as null in successData
```

Reply:
I think the problem is not when we reconnect (because it is working), but when we reopen the app.

What about this?
`// We need success data in case we don't receive the push notification of onyx update.`

The opposite version of this comment here
https://github.com/Expensify/App/blob/3d17dcf7a4fed31c0eef0ba684b97ee1a54396a7/src/libs/actions/Policy.js#L101-L104


(Actually, I am not even sure if we should add a comment in the first place because we already have a guide for this offline pattern [here](https://github.com/Expensify/App/blob/main/contributingGuides/OFFLINE_UX.md#b---optimistic-with-feedback-pattern))

Reply:
> I think the problem is not when we reconnect (because it is working)

It isn't working for me (without the changes). If I go offline and delete the bank account then on reconnect the bank account disappears and I see a blank page (the original issue)

Reply:
I think it's fine to use the suggested comment 
cc: @jasperhuangg for thoughts.

Reply:
Okay cool. Updated the comment. Thanks for your help!

@jasperhuangg @thesahindia